config/locales/cy/claim_review.cy.yml

```diff
@@ -0,0 +1,137 @@
+        date_of_birth: Dyddiad geni
+        address: Cyfeiriad
+
+      additional_claimants_upload:
+        group_claim: "Hawliad grŵp "
+        file_name: Enw Ffeil CSV
+        number_claimants: Nifer o hawlwyr ychwanegol
+
+      representative:
+        has_representative: Cynrychiolydd
+        type: Math o gynrychiolydd
+        organisation_name: Enw'r sefydliad
+        name: Enw llawn
+        address: Cyfeiriad
+        telephone_number: Ffôn
+        mobile_number: Ffôn Symudol
+        email_address: E-bost
+        dx_number: Rhif DX
+        contact_preference: Dull cyswllt dewisol
+
+      respondent:
+        name: Enw
+        address: Cyfeiriad
+        telephone_number: Ffôn
+        employed_by_employer: Cael fy nghyflogi gan y cyflogwr
+        acas_early_conciliation_certificate_number: Rhif Acas
+        work_address: Cyfeiriad gwaith
+
+      additional_respondents:
+        additional_respondents: Atebwyr ychwanegol
+        name: Enw
+        address: Cyfeiriad
+        acas_early_conciliation_certificate_number: Rhif Acas
+
+      employment:
+        job_title: Swydd
+        start_date: Dyddiad cychwyn
+        average_hours_worked_per_week: Oriau gwaith bob wythnos ar gyfartaledd
+        gross_pay: Tâl cyn treth
+        net_pay: Tâl ar ôl treth
+        enrolled_in_pension_scheme: Cynllun pensiwn
+        benefit_details: Manylion buddion
+        current_situation: Sefyllfa gyfredol
+        end_date: Dyddiad gorffen
+        worked_notice_period_or_paid_in_lieu: Cyfnod rhybudd
+        notice_period_end_date: Dyddiad gorffen y cyfnod rhybudd
+        notice_period_pay: Tâl cyfnod rhybudd
+        new_job: Swydd arall
+        new_job_gross_pay: "Tâl cyn treth yn y swydd newydd "
+        new_job_start_date: Dyddiad cychwyn y swydd newydd
+        pay_period_weekly: bob wythnos
+        pay_period_monthly: bob mis
+        was_employed: Cael fy nghyflogi gan y cyflogwr
+
+      claim_type:
+        types: Math(au)
+        is_whistleblowing: Chwythu'r chwiban
+        is_protective_award: "Dyfarniad Gwarchodol "
+        send_claim_to_whistleblowing_entity: Anfon i gorff chwythu'r chwiban
+
+      claim_details:
+        claim_details: Manylion yr hawliad
+        other_known_claimant_names: Hawlwyr hysbys eraill
+        miscellaneous_information: Gwybodaeth arall
+        attached_document: Dogfennau sydd ynghlwm
+
+      claim_outcome:
+        desired_outcomes: "Pa ganlyniad?"
+        other_outcome: Manylion y canlyniad
+
+      additional_information:
+        miscellaneous_information: Manylion pwysig
+
+      your_fee:
+        seeking_remission: "Help i dalu ffioedd "
+        fee: Eich ffi
+
+    email_addresses:
+      submit_warning_header: Cyflwyno eich hawliad
+      submit_warning: "Hwn yw'r cyfle olaf i chi newid manylion eich hawliad ar-lein. Gwiriwch fod yr holl wybodaeth uchod yn gywir. "
+      confirmation: Cadarnhad drwy e-bost
+      submit_with_remission: "Cyflwyno'r hawliad "
```

Comment:
Have you noticed the space at the end ?  Ive spotted this in a few places

Reply:
@garytaylor  well spotted. Removing now.